Docs: API Doc hackathon project (#209)
grafana/writers-toolkit: docs/sources/style-guide/style-conventions/index.md

- Don't use buzzwords or jargon.
- Keep paragraphs to three sentences or less.
  - Condense the text, add more headings, or do both.

> - Use contractions in most cases to express an informal style.

Reviewer: ```suggestion
- Use contractions in common and negative cases: _you’re_, _that’s_, _isn’t_ or _don’t, for example.
```
I prefer to give more specific guidance than _most_.

Reviewer: The style has been more _conversational_ than _informal_, because we advice folks to "Write as though you are having a conversation about technology with a friend."

The most common admonition is a note. A note provides additional information that the user should be aware of.

> For example:

Reviewer 2: Why are you removing the code syntax for creating an admonition (Line 121)?  I _**just**_ put this into the Writer's Handbook after one of my engineers asked me how we create admonitions and I realized that you cannot figure it out without going into the code.  And this repo is NOT public.

Author: Weird. I didn't intentionally remove _anything_.  I also created the working copy on my computer with a fresh repo and all-new clone of everything as of a few days ago. I can't think of any reason why your changes would not be here. Let me look into it further.

Reviewer 2: That is weird!

Author: Perhaps I deleted it unintentionally? I don't know. Added it back, will push soon.

Author: This is fixed in latest file.

A caution warns the user to proceed with caution. A caution emphasizes a course of action's potential downsides.

The markdown syntax for a caution is `> **Caution**` and will appear in the published output as shown in the following example:

> **Caution:** By disabling authentication requirements, anyone can access your Grafana instance. There is a considerable security risk associated with this.

### Warnings

> A warning informs the user not to do something.  Warnings are usually reserved for actions that, if performed, could cause harm to hardware, software, or data.  

Reviewer 3: Why remove this?

Reviewer 2: +1.  My last job had way too many warnings in the documentation because engineers did not understand that there is a standard for when to use them.

Author: Fixed in latest commit

> A warning informs the user not to do something For example:

Reviewer: ```suggestion
A warning informs the user not to do something.

For example:
```

Reviewer: Parallel structure to line 22.

> > **Warning:** You cannot back up your dashboards in Grafana. You might not be able to recover a dashboard if it is deleted.

Reviewer: ```suggestion
> **Warning:** Do not back up your dashboards in Grafana. You might not be able to recover a dashboard if it is deleted.
```
It might be possible, but iladvised.

Reviewer: This warning seems to contradict itself somehow.

Reviewer: What am I supposed to do instead?

> > **Warning:** You cannot back up your dashboards in Grafana. You might not be able to recover a dashboard if it is deleted.

Reviewer 3: ```suggestion
> **Warning:** You cannot back up your dashboards in Grafana. You might not be able to recover a dashboard if you delete it.
```
